src/main/ruby/post-boot/post-boot.rb

```diff
@@ -82,3 +85,9 @@
     end
   end
 end
+
+end
+
+Truffle.privately do
+  Object.remove_method :delay
+end
```

Comment:
This could be written like:
```ruby
class Object
  remove_method :delay
end
```
Any opinion?

Reply:
I like what you have in the code already. It looks more declarative to me.

Reply:
This is gone with moving `delay` under `Truffle::Boot`, there is no need to remove the method anymore.